wqflask/wqflask/correlation/show_corr_results.py

```diff
@@ -484,30 +484,31 @@ def generate_corr_json(corr_results, this_trait, dataset, target_dataset, for_ap
         if trait.view == False:
             continue
         results_dict = {}
-        if not for_api:
-            results_dict['checkbox'] = "<INPUT TYPE='checkbox' NAME='searchResult' class='checkbox trait_checkbox' style='padding-right: 0px;' VALUE='" + hmac.hmac_creation('{}:{}'.format(trait.name, trait.dataset.name)) + "'>"
-            results_dict['index'] = i + 1
-            results_dict['trait_id'] = "<a href='/show_trait?trait_id="+str(trait.name)+"&dataset="+str(dataset.name)+"'>"+str(trait.name)+"</a>"
-        else:
-            results_dict['trait_id'] = trait.name
+        results_dict['index'] = i + 1
+        results_dict['trait_id'] = trait.name
+        results_dict['dataset'] = trait.dataset.name
         if target_dataset.type == "ProbeSet":
             results_dict['symbol'] = trait.symbol
+            if len(trait.description_display) > 40:
+                results_dict['description'] = trait.description_display[:40] + "..."
+            else:
+                results_dict['description'] = trait.description_display
             results_dict['description'] = trait.description_display
             results_dict['location'] = trait.location_repr
-            results_dict['mean'] = float(trait.mean)
+            if trait.mean and trait.mean != "":
```

Comment:
**suggestion:nitpick:** You could cast that into a bool:

``` python
#+begin_src python :results output
print(bool(""))

print(bool(None))

print(bool("some value"))

print(bool(2))
#+end_src

#+RESULTS:
: False
: False
: True
: True
```

So this should work:
```suggestion
            if bool(trait.mean) 
```